pkg/netpol/connlist/connlist_test.go

```diff
@@ -7,19 +7,15 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/np-guard/netpol-analyzer/pkg/netpol/common"
 	"github.com/np-guard/netpol-analyzer/pkg/netpol/internal/testutils"
 	"github.com/np-guard/netpol-analyzer/pkg/netpol/scan"
 
 	"github.com/stretchr/testify/require"
 )
 
 func getConnlistFromDirPathRes(stopOnErr bool, path string) (*ConnlistAnalyzer, []Peer2PeerConnection, error) {
-	var analyzer *ConnlistAnalyzer
-	if stopOnErr {
-		analyzer = NewConnlistAnalyzer(WithStopOnError())
-	} else {
-		analyzer = NewConnlistAnalyzer()
-	}
```

Comment:
let's keep `WithStopOnError()` as it was before, I think we should avoid changing this api interface, as it makes sense to be called with no args (when this option is required).